scripts/deploySidechain.ts

```diff
@@ -52,18 +46,24 @@ export interface CanonicalPhaseDeployed {
 
 export async function deployCanonicalPhase(
     hre: HardhatRuntimeEnvironment,
+    deployer: Signer,
     config: ExtSystemConfig,
     phase2: Phase2Deployed,
     phase6: Phase6Deployed,
-    deployer: Signer,
     debug: boolean = false,
     waitForBlocks: number = 0,
 ): Promise<CanonicalPhaseDeployed> {
+    // -----------------------------
+    // Post:
+    //         Deployer : l1Coordinator.transferOwnership(multisigs.daoMultisig);
+    //     Protocol DAO : l1Booster.setBridgeDelegate(l1Coordinator.address);
+    // -----------------------------
+
     const auraProxyOFT = await deployContract<AuraProxyOFT>(
         hre,
         new AuraProxyOFT__factory(deployer),
         "AuraProxyOFT",
-        [config.lzEndpoint, phase2.cvx.address, phase2.cvxLocker.address],
+        [config.l1LzEndpoint, phase2.cvx.address, phase2.cvxLocker.address],
```

Comment:
just call these lzEndpoint,  no need to change them to `L1`, `L2` as that comes from the context of the config.